guides/screen-reader-differences.md

```diff
@@ -0,0 +1,126 @@
+In this guide:
+
+- [Screen reader overview](#Overview)
+  - [Commonalities](#Commonalities)
+  - [Differences](#Differences)
+      - [Interaction modes](#Interaction-modes)
+- [Lessons](#lessons)
+
+# Overview
+
+Screen reader software allows users to navigate webpages by having the text of the page read aloud to them linearly. Screen reader users can also interact with the elements on the page using special keyboard commands that are unique to each application. The top 3 screen readers according to the [2017 WebAIM Survey](https://webaim.org/projects/screenreadersurvey7/) are JAWS, NVDA, and VoiceOver:
+
+|  Screen Reader |  % of respondents who commonly use it | 
+|---|---|
+|  JAWS | 66.0%  |
+|  NVDA | 64.9%  |
+|  VoiceOver | 39.6%  |
+
+JAWS and NVDA are Windows applications. VoiceOver is available as a desktop Mac OS application and on iOS devices.
+
```

Comment:
Not required, but it may be useful to link to a video of a user using a screen reader. This is a good one: https://www.youtube.com/watch?v=q_ATY9gimOM